packages/apollo-federation/src/composition/__tests__/composeAndValidate.test.ts

```diff
@@ -328,7 +328,12 @@ describe('value types integration tests', () => {
       name: 'serviceB',
     };
 
-    const { errors } = composeAndValidate([serviceA, serviceB]);
+    const { errors, schema } = composeAndValidate([serviceA, serviceB]);
+
+    // A value type doesn't need any federation metadata to resolve
+    const couchType = schema.getType('Couch');
+    expect(couchType.federation.serviceName).toEqual(null);
```

Comment:
I don't think this test is all that valuable. Even if change this implementation internally, what we want to test is the success of the query planner using value types. Tests like these can lock us into an implementation detail that doesn't impact the overall behavior we want to test

Reply:
That makes sense. Agree with testing success of query planner, but I think that belongs elsewhere in the integration tests. Instead, I've added tests for each `Kind` of value type and ensure that they end up in the resulting schema, which would be representative of a successful `composeAndValidate` without testing impl details.

Resolved in 3818c6b